service_manifest.yml

```diff
@@ -69,7 +70,7 @@ heuristics:
   - heur_id: 6
     name: Office password removed
```

Comment:
A bit long.  Heuristic 10 is:  Extracted from Protected Archive, so perhap  "Extracted from Protected office document"